app/views/service_calendars/_update_service_calendar.js.coffee

```diff
@@ -1,24 +0,0 @@
-# Copyright © 2011-2016 MUSC Foundation for Research Development
```

Comment:
File removed, replaced by `toggle_calendar_column.js.coffee` and `toggle_calendar_row.js.coffee`